spec.md

```diff
@@ -282,9 +291,9 @@ The request provides these query string parameters as useful hints for brokers.
 |---|---|---|
 | service_id  |  string | ID of the service from the catalog.  |
 | plan_id  | string  | ID of the plan from the catalog.  |
-| operation  |  string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client should provide the same value using this query parameter as a URL-encoded string.  |
+| operation  |  string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client SHOULD provide the same value using this query parameter as a URL-encoded string.  |
```

Comment:
Well, there's two things here at play so let's make sure we figure out which one we're talking about:
 1. Broker SHOULD/MUST return a last_operation in the response to async provisioning request
 2. Platform SHOULD/MUST provide the same value

@pmorie  you and I were discussing #1 because it would simplify writing the state management in the controller. So just wanted to make sure we're talking about the right 'this' :)